ricecooker/managers/tree.py

```diff
@@ -144,6 +144,10 @@ def get_file_diff(self, files_to_diff):
                 if not exists
             ]
 
+class InsufficientStorageException(Exception):
```

Comment:
These seem like reasonable changes. Please remove changes to the other files.

If you would like to add the `audio_cache.sqlite` file to the `.gitignore` that would be much appreciated.